docs/install-development-cluster.md

```diff
@@ -0,0 +1,30 @@
+# Installing in a Development Environment
+
+> [!NOTE]
+> This instalation is meant for testing purposes only. It uses self signed certificates, so your browser will show a warning about it.
+
+To install Tracetest OnPrem in a development environment, and get a first glimpse of how it works, you can follow the instructions below.
+
+We provide a development install via a script that you can use to run Tracetest OnPrem locally on your machine. The only prerequisite is that you have [kind](https://kind.sigs.k8s.io/) installed.
+
+You can then do a `git checkout` of the helm chart repo and follow the instructions when needed:
+```sh
+cd ./my-folder
+
```

Comment:
```suggestion
```

I'd remove those lines, it assumes `my-folder` exists and adds an extra depth level to the dir structure